lib/static/new-ui/features/suites/components/TestStatusFilter/index.tsx

```diff
@@ -0,0 +1,46 @@
+import {RadioButton} from '@gravity-ui/uikit';
+import React, {ReactNode} from 'react';
+import {connect} from 'react-redux';
+import {bindActionCreators} from 'redux';
+
+import {TestStatus, ViewMode} from '@/constants';
+import * as actions from '@/static/modules/actions';
+import {getStatusCounts, StatusCounts} from '@/static/new-ui/features/suites/components/TestStatusFilter/selectors';
+import {State} from '@/static/new-ui/types/store';
+import {getIconByStatus} from '@/static/new-ui/utils';
+import styles from './index.module.css';
+
+interface TestStatusFilterOptionProps {
+    status: TestStatus | 'total';
+    count: number;
+}
+
+function TestStatusFilterOption(props: TestStatusFilterOptionProps): ReactNode {
+    return <div className={styles['test-status-filter-option']}>
+        {props.status === 'total' ? <span>All</span> : getIconByStatus(props.status)}<span className={styles['test-status-filter-option__count']}>{props.count}</span>
+    </div>;
+}
+
+interface TestStatusFilterProps {
+    statusCounts: StatusCounts;
+    actions: typeof actions;
+    viewMode: ViewMode;
+}
+
+function TestStatusFilterInternal({statusCounts, actions, viewMode}: TestStatusFilterProps): ReactNode {
+    return <RadioButton width={'max'} onChange={(e): void => void actions.changeViewMode(e.target.value)} value={viewMode}>
+        <RadioButton.Option value={ViewMode.ALL} content={<TestStatusFilterOption status={'total'} count={statusCounts.total}/>} />
```

Comment:
i would like to add a title for each option (but i'm not sure that it's possible for this component)

Reply:
Done!
Looks like this now:
![](https://i.imgur.com/H5aJJTS.png)